MANIFEST.in

```diff
@@ -1,10 +1,18 @@
 include README.rst
 include LICENSE
 include MANIFEST.in
+include NEWS
 include scrapyd/VERSION
 include scrapyd/default_scrapyd.conf
-recursive-include scrapyd/tests *.egg
-recursive-include scrapyd/scripts *.py
+recursive-include scrapyd *.py
 recursive-include docs *
+recursive-include bin *
 prune docs/_build
 recursive-include extras *
+recursive-exclude debian *
+exclude .bumpversion.cfg
+exclude .coveragerc
+exclude *.buildbot
+include tox.ini
```

Comment:
Looks like tox.ini is deleted?

Reply:
good catch thanks fixed https://github.com/scrapy/scrapyd/pull/412/commits/50e47f26a7682aaefc81737d18b0b332c7d52bfb